src/scripts/run-tests.sh

```diff
@@ -11,26 +20,36 @@ unzip -qod "${tmpdir}/scriptgen" "${tmpdir}/scriptgen.zip"
 # form OS appropriate paths for MATLAB
 os=$(uname)
 gendir=$tmpdir
+workdir=$(pwd)
+binext=""
 if [[ $os = CYGWIN* || $os = MINGW* || $os = MSYS* ]]; then
     gendir=$(cygpath -w "$gendir")
+    workdir=$(cygpath -w "$workdir")
+    binext=".exe"
 fi
 
 # generate and run MATLAB test script
-"${tmpdir}/bin/run_matlab_command.sh" "\
-    addpath('${gendir}/scriptgen');\
-    testScript = genscript('Test',\
-    'JUnitTestResults','${PARAM_TEST_RESULTS_JUNIT}',\
-    'CoberturaCodeCoverage','${PARAM_CODE_COVERAGE_COBERTURA}',\
-    'HTMLCodeCoverage','${PARAM_CODE_COVERAGE_HTML}',\
-    'SourceFolder','${PARAM_SOURCE_FOLDER}',\
-    'SelectByFolder','${PARAM_SELECT_BY_FOLDER}',\
-    'SelectByTag','$PARAM_SELECT_BY_TAG',\
-    'CoberturaModelCoverage','${PARAM_MODEL_COVERAGE_COBERTURA}',\
-    'HTMLModelCoverage','${PARAM_MODEL_COVERAGE_HTML}',\
-    'SimulinkTestResults','${PARAM_TEST_RESULTS_SIMULINK_TEST}',\
-    'HTMLTestReport','${PARAM_TEST_RESULTS_HTML}',\
-    'PDFTestReport','${PARAM_TEST_RESULTS_PDF}');\
-    disp('Running MATLAB script with contents:');\
-    disp(testScript.Contents);\
-    fprintf('__________\n\n');\
-    run(testScript);"
+script=command_${RANDOM}
+scriptpath=${tmpdir}/${script}.m
+echo "cd('${workdir//\'/\'\'}');" > "$scriptpath"
+cat << EOF >> "$scriptpath"
```

Comment:
What's the idea with the move to heredoc?

Reply:
It just seemed more consistent with how we run MATLAB from other tools and what we're recommending in other tools. I'm open to putting it back to how it was.

Reply:
I think it's OK. Just don't love that we have to write another file to disk and cd around more.

Using heredoc, you should be able to get rid of the `\` at the end of each line and actually make it a multi-line script (the multi-line MATLAB commands need `...` line continuations though). Some indentation might help readability too.